Added lstm model and input data for adcirc_nn test

west5678 opened the pull request.

west5678 committed:
Added lstm model and input data for adcirc_nn test

gajanan-choudhary committed:
Merge branch 'main' into wei/lstm-test

This is being done to update the test case in wei's branch to the latest
one which has water surface elevation boundary conditions.

gajanan-choudhary commented:
@west5678, does the `nn_input` folder have to be in a separate directory the way you have currently structured it? I was planning to keep each test-case folder as standalone and independent of other files/directories, even if we have some repeated files. Would it be okay with you if I move all the files in the `nn_input/` directory into the `large` test case directory?

west5678 commented:
> @west5678, Would it be okay with you if I move all the files in the `nn_input/` directory into the `large` test case directory?

I think it is OK. The functions I wrote could specify a directory of the input files. So we can simply change it in the lstmnn.py.

gajanan-choudhary committed:
Increase 'large' simulation duration to 4 days

gajanan-choudhary committed:
Move 'nn_input' dir into 'large' test case dir

This is needed for future proofing. We want all test cases to be
standalone and independent of other tests. This may cause some files to
be repeated across test cases, but that is fine.

gajanan-choudhary reviewed (approved):
Approved.
